### PR TITLE
fix-search.vue

### PR DESCRIPTION
**Pull request by NaelaTarannum**

### CODE REVIEW

**Comment by NaelaTarannum** [comment]
solves issue #864

File: src/components/General/Search.vue [hunk 1]
```diff
@@ -13,6 +13,7 @@
         @selected="selected"
       />
     </q-search>
+    <span v-if="!results.length" >No results.</span>
```
Comment (nicksellen) [inline src/components/General/Search.vue]: All the text needs to be internationalized so we can manage the translations.

Have a look at the other files for examples of how to do this, e.g. from CurrentUser.vue there is:

```vue
<span>{{ $t('PICKUPLIST.ITEM.LEAVE') }}</span>
```

File: src/components/Group/Navbar.vue [hunk 2]
```diff
@@ -9,6 +9,20 @@
       class="col"
       ref="groupnav"
     >
+      <div class="">
```
Comment (nicksellen) [inline src/components/Group/Navbar.vue]: Can remove attribute if it is empty

File: src/components/Group/Navbar.vue [hunk 3]
```diff
@@ -9,6 +9,20 @@
...
+        <q-transition
+          duration="510"
+          name="turn-in"
+          appear
+        >
+          <div class="photoCard shadow-2">
+            <ProfilePicture
+              :group="group"
```
Comment (tiltec) [inline src/components/Group/Navbar.vue]: The changes in Navbar.vue seem unrelated to the referenced issue. What was your intention here?
Also,`ProfilePicture` doesn't have a `group` prop, but needs a `user` prop, so this code seems unlikely to work.

File: src/components/General/Search.vue (hunk 1 above)
Comment (NaelaTarannum) [reply src/components/General/Search.vue]: Will try

**Comment by tiltec** [comment]
Closing this in favor of #965
